codes/c/chapter_dynamic_programming/min_path_sum.c

```diff
@@ -0,0 +1,125 @@
+/**
+ * File: min_path_sum.c
+ * Created Time: 2023-10-02
+ * Author: Zuoxun (845242523@qq.com)
+ */
+
+#include "../utils/common.h"
+
+int n, m;
+int grid_size, grid0_size;
+
+/* 最小路径和：暴力搜索 */
+int minPathSumDFS(int grid[][m], int i, int j) {
+    // 若为左上角单元格，则终止搜索
+    if (i == 0 && j == 0) {
+        return grid[0][0];
+    }
+    // 若行列索引越界，则返回 +∞ 代价
+    if (i < 0 || j < 0) {
+        return INT_MAX;
+    }
+    // 计算从左上角到 (i-1, j) 和 (i, j-1) 的最小路径代价
+    int left = minPathSumDFS(grid, i - 1, j);
+    int up = minPathSumDFS(grid, i, j - 1);
+    // 返回从左上角到 (i, j) 的最小路径代价
+    return min(left, up) != INT_MAX ? min(left, up) + grid[i][j] : INT_MAX;
```

Comment:
`min` not declared

Reply:
I've declared function min at line 10 in the code, didn't you see it? How about refresh?

Reply:
> I've declared function min at line 10 in the code, didn't you see it? How about refresh?

It didn’t happen before my review, but now it’s ok